CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# How to Contribute to CasterSoundboard
+
+CasterSoundboard is a volunteer-run project, built and maintained by many contributors. We welcome new team members, and value feedback from users using CastrerSoundboard.
+
+The following is a set of guidelines for contributing to CasterSoundboard, which are hosted in the Jupiter Broadcasting Organization on GitHub. These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+### Reporting a Bug Report or Requesting a Feature
+We utilize a template to keep our issue tracker organized. Please fill it out and remove areas that are not related to your Bug Report or Feature Request. Also keep these guidelines in mind as well:
+
+* Search before posing to avoid duplicates
+* Please keep the topic of the issue or feature request to **ONE** problem or request.
```

Comment:
Mostly agree with this, the only issue (pun intended) is that this project is still in it's early stages so it's likely that feature requests could be quite broad in scope. For example, issue #53 - I guess the title could be better but it seems like it's being used for throwing around ideas about audio filters in general, rather than a single feature (unless you were to count audio filters as a single feature... hmmm 🤔).

Maybe just un-bold "**ONE**" or something to make it a little less concrete for a while?